hibernate-core/src/test/java/org/hibernate/id/idclass/MyEntityId.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.id.idclass;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class MyEntityId implements Serializable {
```

Comment:
It would be good practise to use package default visibility (remove `public` above) for these classes are only used by the testing case within the same package.

Reply:
I prefer leaving this as-is